src/public/bridge/windowbridge.cpp

```diff
@@ -31,4 +31,41 @@ int32_t WindowGetPosY(void) {
 bool WindowIsFullscreen(void) {
     return Ship::Context::GetInstance()->GetWindow()->IsFullscreen();
 }
+
+/**
+ * Widescreen functions
+ *
+ */
+
+float ScreenGetAspectRatio() {
+    return gfx_current_dimensions.aspect_ratio;
+}
+
+// AdjustXFromLeftEdge 
+float LeftEdgeAlign(float v) {
+    return (SCREEN_WIDTH / 2 - SCREEN_HEIGHT / 2 * OTRGetAspectRatio() + (v));
```

Comment:
These `SCREEN_WIDTH` and `SCREEN_HEIGHT` should be replaced with `gfx_native_dimensions.width` and `gfx_native_dimensions.height` respectively.

This is necessary for games that change resolutions mid-game (mm with the bombers notebook).